docker-app/qfieldcloud/urls.py

```diff
@@ -70,6 +73,15 @@
         ),
         name="project_file_download",
     ),
+
+      # New team-related endpoints
+    path("organizations/<str:organization_name>/team/", TeamListCreateView.as_view(), name='team-list-create'),
+    path("organizations/<str:organization_name>/team/<str:team_name>/members/", TeamMemberView.as_view(), name='team_members'),
+    path("organizations/<str:organization_name>/team/<str:team_name>/", TeamDetailView.as_view(), name='team-detail'),
+    path("organizations/<str:organization_name>/team/<str:team_name>/members/<str:member_username>/", TeamMemberDeleteView.as_view(), name='team-member-delete'),
+   
+
+
```

Comment:
This should be moved to `qfieldcloud.core.url` not here in `qfieldcloud.url`. If you keep it here, these endpoints are not going to be versioned with `api/v1/`. This file is for global endpoints, not really DRF endpoints.

Reply:
Direct urls moved to core urls. Only global left here